### PR TITLE
replace process.nextTick with queueMicrotask

### PR DESCRIPTION
**Pull request by ftreesmilo**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

**Which issue (if any) does this pull request address?**

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

**Comment by ftreesmilo** [comment]
#1568

File: index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const path = require('path')
 const Peer = require('simple-peer')
 const randombytes = require('randombytes')
 const speedometer = require('speedometer')
+const queueMicrotask = require('queue-microtask');
```
Comment (KayleePop) [inline index.js:16]: tests are failing because of this semicolon :stuck_out_tongue_winking_eye:

Comment (ftreesmilo) [reply index.js:16]: argh

**Review by KayleePop — APPROVED** [review]
:+1: thanks!

**Pull request merged**